specs/exe-script.yaml

```diff
@@ -0,0 +1,153 @@
+openapi: 3.0.2
+info:
+  version: 0.1.0
+  title: Yagna ExeScript
+  description: '
+    The specification of ExeScript commands format as per [Activity API specification]
+    (https://docs.google.com/document/d/1BXaN32ediXdBHljEApmznSfbuudTU8TmvOmHKl0gmQM)
```

Comment:
I'd not link it here since it is not a public doc. We need to place a synthesis/brief of it here

Reply:
Agreed, removed the link.